front-end/src/views/calls/GroupsView.vue

```diff
@@ -19,6 +21,7 @@ export default {
 // and pass it to the QrCode component
 </script>
 <template>
+	<ReturnButtonComponent />
 	<StudentListComponent />
 	<GroupsComponent />
 	<Chat room-id="1" />
```

Comment:
Is the room-id always supposed to be 1 ?

Reply:
I guess it should be because it's a default value for the room.